server-spi/src/main/java/org/keycloak/services/clientpolicy/ClientPolicyManager.java

```diff
@@ -73,7 +73,6 @@ public interface ClientPolicyManager extends Provider {
      * when getting client profiles via Admin REST API, returns the existing client profiles set on the realm.
      * 
      * @param realm - the realm whose client profiles is to be returned
-     * @param includeGlobalProfiles - If true, method will return realm profiles and global profiles as well. If false, then "globalProfiles" field would be null
```

Comment:
Nitpick: Not sure why this line is removed? The param `includeGlobalProfiles` is still applied for the method `getClientProfiles` if I understand correctly?

Reply:
Nah! It was a mistake. Probably I wanted to copy the line to the policies method and I cut it instead. Thanks for spotting it. Re-added the line.